float64 linear_x #the x partition of linear_speed
float64 linear_y #the y partition of linear_speed
